app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'csv'
 require 'reporting/proofing_rate_report'
+require 'reporting/monthly_idv_report'
```

Comment:
I would expect this to be an extension of the Proofing Rate Report? The Proofing Rate Report is the IDV report sliced over a different time period